aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -36,9 +36,9 @@ module aptos_framework::aptos_governance {
     const EALREADY_VOTED: u64 = 4;
     const ENO_VOTING_POWER: u64 = 5;
 
-    /// Store the SignerCapability of the framework account (0x1) so AptosGovernance can have control over it.
+    /// Store the SignerCapabilities of accounts under the on-chain governance's control.
     struct GovernanceResponsbility has key {
-        signer_cap: SignerCapability,
+        signer_caps: Table<address, SignerCapability>,
```

Comment:
I think this as a SimpleMap would probably suffice

Reply:
Updated